### PR TITLE
added 4gb warning to docker compose instructions

### LINKED ISSUE #222: Fix Get Started documentation
The `Get Started` instructions for setting up a docker based OpenSearch cluster does not work as mentioned. These instructions are missing some pre-requisite settings for linux and Mac, which make docker setup fail. This creates confusion and bad Getting started experience for users. 

Fix is suggested here - https://github.com/opensearch-project/OpenSearch/issues/978

### PR DESCRIPTION
**Pull request by stockholmux**
### Description
Adds instruction to set docker to use a larger amount of ram for the example
 
### Issues Resolved
#222

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _includes/downloads/opensearch-docker.markdown [hunk 1]
```diff
@@ -1,7 +1,8 @@
 The best way to try out OpenSearch is to use [Docker Compose](https://docs.docker.com/compose/install/). These steps will setup a two node cluster of OpenSearch plus OpenSearch Dashboards:
 
 1. Download [docker-compose.yml](https://opensearch.org/samples/docker-compose.yml) into your desired directory
-2. Run `docker-compose up`
-3. Have a nice coffee while everything is downloading and starting up
-4. Navigate to [http://localhost:5601/](http://localhost:5601) for OpenSearch Dashboards
-5. Login with the default username (`admin`) and password (`admin`)
+2. Make sure you have Docker set to use at least 4 GB of RAM in **Preferences** > **Resources**.
```
Comment (dblock) [inline _includes/downloads/opensearch-docker.markdown:4]: Is this Windows only? Desktop version? How do I do this on Linux command line?

Comment (stockholmux) [reply _includes/downloads/opensearch-docker.markdown:4]: This is exactly how it's stated in documentation - if we want to change this line we should go docs first.

Comment (owaiskazi19) [reply _includes/downloads/opensearch-docker.markdown:4]: The above step is only for Windows/OSX user. @stockholmux you missed for Linux/Debian users as mentioned here https://github.com/opensearch-project/OpenSearch/issues/978#issuecomment-885228338

Comment (stockholmux) [reply _includes/downloads/opensearch-docker.markdown:4]: @owaiskazi19 See https://github.com/opensearch-project/OpenSearch/issues/978#issuecomment-885252386

**Pull request merged**